python/ray/autoscaler/autoscaler.py

```diff
@@ -53,6 +53,10 @@
     # The number of workers to launch initially, in addition to the head node.
     "initial_workers": (int, OPTIONAL),
 
+    # Whether or not to scale aggressively, e.g. to jump back to at least
+    #   initial_workers if we're ever below it and are scaling up
+    "aggressive_autoscaling": (bool, OPTIONAL),
```

Comment:
In the anticipation of further modes, how about `"mode": "aggressive"`?

Reply:
@ls-daniel This should be resolved before we merge this. I agree with @ericl's suggestion.

Reply:
This is the only outstanding comment I think?

Reply:
Oops. Right, I'll change this soon, my ability to test changes is broken at the moment.